lib/build/sphinx_ext.py

```diff
@@ -213,7 +211,7 @@ def _BuildOpcodeResult(op_id):
   return "``%s``" % result_fn
 
 
-class OpcodeParams(s_compat.Directive):
+class OpcodeParams(docutils.parsers.rst.Directive):
```

Comment:
I think it's more readable to just once `from docutils.parsers.rst import Directive` and then use plain `Directive` for all classes.

Reply:
The file did already import docutils.parsers.rst and use it like that, but I agree that this looks more readable. Is changed.